AI/app/data/util/plan_converter.py

```
@@ -0,0 +1,189 @@
+                "addr2": row.addr2 or "",
+                "categoryId": row.category_id,
+                "categoryName": row.category_name,
+                "date": str(row.pa_date),
+                "attractionOrder": row.pa_attraction_order
+            }
+            plan_dict[row.plan_id].append((row, attraction_info))
+        else:
+            # 여행지 없는 일정도 있을 수 있음
+            plan_dict[row.plan_id].append((row, None))
+
+    for plan_id, items in plan_dict.items():
```

Ruff (0.11.9)

Loop control variable `plan_id` not used within loop body

Rename unused `plan_id` to `_plan_id`

(B007)